app/src/main/res/layout/link_id_with_patient_view.xml

```diff
@@ -24,8 +24,7 @@
     android:background="?attr/colorSurface"
     android:elevation="20dp"
     app:cardCornerRadius="4dp"
-    app:contentPaddingBottom="4dp"
-    tools:ignore="UnusedAttribute">
+    app:shapeAppearanceOverlay="">
```

Comment:
Was this change meant to be part of the next commit?

Reply:
Ah, yes. My bad, I can discard this if you want.

Reply:
Fixed it